commands/poll.js

```diff
@@ -0,0 +1,151 @@
+const Discord = require('discord.js');
+const emojiRegex = require('emoji-regex');
+
+const symbols = require('../symbols');
+const functions = require('../functions');
+
+function generateTestResults(options) {
```

Comment:
I like doc comments, I have no idea what this does by looking at it. And the fact that it's at the top of the file...

I like having the first main function first at the top of the file and then
the functions that get called after it after the function in order of being called.

Reply:
Yep I definitely need more comments dw you got me there.